### ISSUE TITLE
Clicking thumbnail on File Detail throws permission error

### ISSUE THREAD

**Issue by benhaynes** [RNGR/Directus#823]
For some reason, if you click on the thumbnail on a "File Detail" page (see below) it throws a permission error. It has a target=blank... so it should open in a new window (doesn't) and the image is viewable if you right-click and say open in new tab – so it doesn't seem to be permission based. Could it be the routing?

```
    <a target="_BLANK" href="http://admin.xxxxx.com/media/00000009980.jpg">
        <div class="ui-thumbnail has-file">
            <img src="http://admin.xxxxx.com/media/thumbs/9980.jpg">
        </div>
    </a>
```

![screen shot 2015-06-18 at 2 45 31 pm](https://cloud.githubusercontent.com/assets/522079/8239371/5db769e8-15c9-11e5-8da7-dffb96d7ca9a.png)

**Clicking the above, gives you below:**

![screen shot 2015-06-18 at 2 46 45 pm](https://cloud.githubusercontent.com/assets/522079/8239369/5a05425c-15c9-11e5-8b2b-6c0d4607752f.png)

**Comment by WellingGuzman** [comment]
This was fixed by #824

**Comment by WellingGuzman** [comment]
This was solved already, I'd close it.

Let me know if this still happening to you.
